File: example-databinding/src/main/java/com/xwray/groupie/example/databinding/item/ColumnItem.java
package com.xwray.groupie.example.databinding.item;

import android.support.annotation.ColorInt;

import com.xwray.groupie.example.databinding.MainActivity;

public class ColumnItem extends CardItem {

    public ColumnItem(@ColorInt int colorRes, int index) {
        super(colorRes, String.valueOf(index));
        getExtras().put(MainActivity.INSET_TYPE_KEY, MainActivity.INSET);
    }

    @Override public int getSpanSize(int spanCount, int position) {
        return spanCount / 2;
    }
}

File: example/src/main/java/com/xwray/groupie/example/item/ColumnItem.kt
package com.xwray.groupie.example.item

import androidx.annotation.ColorRes
import com.xwray.groupie.example.INSET
import com.xwray.groupie.example.INSET_TYPE_KEY

class ColumnItem(@ColorRes colorRes: Int, index: Int) : CardItem(colorRes, index.toString()) {

    init {
        extras[INSET_TYPE_KEY] = INSET
    }

    override fun getSpanSize(spanCount: Int, position: Int) = spanCount / 2

}
